src/doc/language.ts

```diff
@@ -1562,6 +1690,15 @@ export const doc_language:Documentation = {
 				only in the first case the result is an integer. The corresponding
 				overflow rules apply.
 				</p>
+                <div class="example">
+                <h3>Example</h3>
+                <tscript do-not-run>
+print(4 - 3);                   # prints 1
+print(3.5 - 2.5);               # prints 1.0
```

Comment:
No, it prints `1`. Also, `print(1.0);` prints `1`. The output is not marked explicitly as a real, since it is for the user, not for the programmer.